### PR TITLE
Seismic misfit improvements

### PR DESCRIPTION
**Pull request by rnyb**
Improved performance when using polygon files with a large number of polygons.
Also changed default value of sim_path.

### CODE REVIEW

File: webviz_subsurface/plugins/_seismic_misfit.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def __init__(
         webviz_settings: WebvizSettings,
         ensembles: List[str],
         attributes: List[str],
-        attribute_sim_path: str = "sim2seis/output/4d_attribute_maps/",
+        attribute_sim_path: str = "share/results/maps/",
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_seismic_misfit.py:164]: Is this a new default output path? Do we need to do any changes in `testdata` or is it ok as is?

File: webviz_subsurface/plugins/_seismic_misfit.py [hunk 2]
```diff
@@ -2109,9 +2109,9 @@ def update_misfit_plot(
             x="REAL",
             y="ABSDIFF",
             title=ens_name,
-            range_y=[min_diff * 0.30, max_diff * 1.05],
+            range_y=[min_diff * 0.0, max_diff * 1.05],
             color="ABSDIFF",
-            range_color=[min_diff * 0.30, max_diff * 1.00],
+            range_color=[min_diff * 0.0, max_diff * 1.00],
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_seismic_misfit.py:2114]: What is the reason for this change? Should it just be `0.0` instead of `min_diff * 0.0` which is anyway 0?

File: webviz_subsurface/plugins/_seismic_misfit.py [hunk 3]
```diff
@@ -2257,21 +2257,20 @@ def update_obsdata_map(
...
-            fig.add_trace(
+        for poly_id, polydf in df_polygon.groupby("POLY_ID"):
+            fig.append_trace(
                 go.Scattergl(
                     x=polydf["X_UTME"],
                     y=polydf["Y_UTMN"],
                     mode="lines",
                     line_color="RoyalBlue",
-                    name=poly_id,
+                    name="pol" + str(poly_id),
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_seismic_misfit.py:2267]:
```suggestion
                    name=f"pol{poly_id}",
```

File: webviz_subsurface/plugins/_seismic_misfit.py [hunk 4]
```diff
@@ -2515,21 +2514,20 @@ def update_obs_sim_map_plot(
...
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_seismic_misfit.py:2524]:
```suggestion
                    name=f"pol{poly_id}",
```

**Review by anders-kiaer — CHANGES_REQUESTED** [review]
LGTM @rnyb. Some small comments below. Also add a changelog entry?

File: webviz_subsurface/plugins/_seismic_misfit.py (hunk 1 above)
Comment (rnyb) [reply webviz_subsurface/plugins/_seismic_misfit.py:164]: Yes, this is a new default output. I recently made a change in testdata webviz config file to prepare for this change so it should work as is. However, we could consider also moving the data itself from `sim2seis/output/4d_attribute_maps` to `share/results/maps/`, but it is not necessary for this to work.

File: webviz_subsurface/plugins/_seismic_misfit.py (hunk 2 above)
Comment (rnyb) [reply webviz_subsurface/plugins/_seismic_misfit.py:2114]: I changed this to 0 as i ran into a case where some realization results were initially hidden, but i agree this was a sloppy change. I will drop the min_diff.

**Review by anders-kiaer — APPROVED** [review]
